command/c/chown.md

```diff
@@ -0,0 +1,282 @@
+# `chown`
+
+## Description
+
+`chown` stands for change owner which changes user or group ownership of a file, directory, or symbolic link. Every file or directory has group or user ownership in Linux. Since multiple users and groups can use same linux system so they all have unique names and IDs. Name and ID of a user and a group can be same. 
+
+For displaying the owner of any file or folder, one can use 
+```shell
+ls -l [file or folder name]
+```
+
+Example: 
+
+```shell
+ls -l test.txt 
+```
+
+_Output:_
+```shell
+-rw-rw-r-- 1 sasa sasa 0 Dec  3 21:42 test.txt
+```
+
+In above output section, first `sasa` denotes user and second `sasa` denotes its group.
+
+
+
+You can also check the owner ship of any folder. Suppose you are in folder which contains test_folder which you can check by using ls command then type below command to see owner ship of your `test_folder`.
+
+Examples: 
+```shell
+ls -l 
+```
+
+_Output:_
+```shell
+drwxrwxr-x  2 sasa sasa    4096 Dec  3 21:42  test_folder
+
+```
+
+### 1. Changing ownership of file to another user.
+
+First checking the ownership of file from below command:
+
+```shell
+ls -l test.txt
+```
+Suppose output is:
+
+_Output:_
+```shell
+-rw-rw-r-- 1 sasa sasa 0 Dec  3 21:42 test.txt
+
+```
+
+Now, Suppose I want to change it to user `root` from user `sasa` then it can be done by below command: 
+
+```shell
+sudo chown root test.txt
+```
+
+_Output:_
+```shell
+-rw-rw-r-- 1 root sasa 0 Dec  3 21:42 test.txt
+
+```
+
+In above output section, ownership changed to user `root` from user `sasa`. Group `sasa` still has ownership. 
+
+_Note:_
+
+You can also use owner `user_id` in place of owner to change ownership in above example like: 
+    
+```shell
+id -u user_name
+```
+Suppose, output is 1000.
+
+Now, you can replace `user_name` with its `user_id`. 
+Example: In place of `sudo chown user_name test.txt`, you can use below:
+
+```shell
+sudo chown 1000 test.txt
+
+```
+
+_Another Note:_
+For changing ownership of folder, you can do it similarly like:
+
+```shell
+sudo chown root test_folder
+```
+It gives ownership of test_folder to `root` user.
+ 
```

Comment:
It is better to organize the two note in list format

Reply:
made it look good

Reply:
done